cmd/minikube/cmd/mount.go

```diff
@@ -110,5 +120,6 @@ var mountCmd = &cobra.Command{
 }
 
 func init() {
+	mountCmd.Flags().StringVar(&mountIP, "ip", "", "specify the ip that the mount should be setup on")
```

Comment:
capitalize first letter of usage?  we had a PR recently to standardize these, so I guess we can continue to follow that